module-9/server/models/User.js

```diff
@@ -0,0 +1,28 @@
+import mongoose from "mongoose";
+
+const UserSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+    email: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    role: {
+      type: String,
+      enum: ["admin", "normal"],
+      default: "normal",
+      required: true,
+    },
+  },
+  {
+    timestamps: true,
+  }
+);
```

Comment:
Make sure that the user model is updated to store the hashed password in the database. This will allow the system to securely retrieve the hashed password during login and compare it with the one provided by the user for authentication.

```suggestion
const UserSchema = new mongoose.Schema(
  {
    name: {
      type: String,
      required: true,
    },
    email: {
      type: String,
      required: true,
      unique: true,
    },
    role: {
      type: String,
      enum: ["admin", "normal"],
      default: "normal",
      required: true,
    },
    password: {
      type: String,
      required: true,
    },
  },
  {
    timestamps: true,
  }
);
```